recipes/recipes_emscripten/lfortran/build.sh

```diff
@@ -0,0 +1,21 @@
+set -e
+set -x
+
```

Comment:
I think we would have liked to call `build0.sh` here (anywhere before `build1.sh`) but then I see the following in the CMakeLists.txt file. 
```
# We don't execute this if we have a tarball
if (LFORTRAN_BUILD_ALL)
    find_program(BASH_BIN bash)
    execute_process(COMMAND "${BASH_BIN}" "-e" "build0.sh" WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} RESULT_VARIABLE _BUILD0_EXIT)
    if (NOT _BUILD0_EXIT EQUAL 0)
        message(FATAL_ERROR "Running build0.sh failed (see error above)")
    endif ()
endif()
```

So as we are dealing with a tarball here I guess we don't need to execute this and hence `LFORTRAN_BUILD_ALL` can be toggled OFF (as done below) and subsequently we don't need to run `build0.sh`. I think that should work !

Reply:
Yes, since we have a tarball, turning off `LFORTRAN_BUILD_ALL` should be fine.